src/clientsettingsdlg.h

```diff
@@ -44,38 +44,40 @@
 #include "multicolorled.h"
 #include "ui_clientsettingsdlgbase.h"
 
-
 /* Definitions ****************************************************************/
 // update time for GUI controls
-#define DISPLAY_UPDATE_TIME         1000 // ms
-
+#define DISPLAY_UPDATE_TIME 1000 // ms
 
 /* Classes ********************************************************************/
 class CClientSettingsDlg : public CBaseDlg, private Ui_CClientSettingsDlgBase
 {
     Q_OBJECT
 
-public:
+  public:
     CClientSettingsDlg ( CClient*         pNCliP,
                          CClientSettings* pNSetP,
                          QWidget*         parent = nullptr );
 
-    void SetStatus ( const CMultiColorLED::ELightColor eStatus ) { ledNetw->SetLight ( eStatus ); }
+    void SetStatus ( const CMultiColorLED::ELightColor eStatus )
+    {
+        ledNetw->SetLight ( eStatus );
+    }
 
     void ResetStatusAndPingLED()
     {
         ledNetw->Reset();
         ledOverallDelay->Reset();
     }
 
-    void SetPingTimeResult ( const int                         iPingTime,
-                             const int                         iOverallDelayMs,
-                             const CMultiColorLED::ELightColor eOverallDelayLEDColor );
+    void SetPingTimeResult (
```

Comment:
No idea what was happening in the old code.  This is better!

Reply:
got reset by `ColumnLimit: 120`